huawei-wmi.c

```diff
@@ -82,6 +82,8 @@ static const struct key_entry huawei_wmi_keymap[] = {
 	{ KE_KEY,    0x289, { KEY_WLAN } },
 	// Huawei |M| key
 	{ KE_KEY,    0x28a, { KEY_CONFIG } },
+	// Ignore Ambient Light Sensoring
+	{ KE_KEY,		 0x2c1, { KEY_RESERVED } },
```

Comment:
This should come last before `{ 0 }`

Reply:
The lines have been moved to the bottom of the list